docker-compose.debug.yml

```diff
@@ -0,0 +1,46 @@
+# 5500 - Frontend
+# 5501 - Backend
+# 5502 - postgres
+# 5503 - redis
+
+# You can initialize the database with data using the "Docker.initdb" image
+# Make sure when creating the container that you specify the --network dofuslab_default switch
+
+version: '3.4'
+
+services:
+  redis:
+    image: "redis:alpine"
+    ports:
+      - "5503:6379"
+
+  postgres:
+    image: "postgres:alpine"
+    environment:
+      POSTGRES_USER: dofuslab
+      POSTGRES_PASSWORD: do-not-use-in-production
+      POSTGRES_DB: dofuslab
+    ports:
+      - "5502:5432"
+    depends_on:
+      - redis
+  
+  server:
+    image: dofuslab-server
+    build:
+      context: ./server
+      dockerfile: Dockerfile
+    ports:
+      - "5501:5501"
+    depends_on:
+      - postgres
```

Comment:
My understanding is that the only thing `depends_on` does is control container startup order, but doesn't do anything like ensure some container like `postgres` is ready to serve in the first place. Is that the case? If so, I feel that these should be removed to avoid giving other devs the wrong idea.

Reply:
you are correct, it could happen that postgres container is running but that postgres is not ready but this could only happen if the redis startup is faster than postgres startup